### PR TITLE
feat: /hardcore command

### PR DESCRIPTION
**Pull request by kenos1**
admittedly a little hacky due to rhinojs limitations

### CODE REVIEW

**Comment by BalaM314** [comment]
✨merge conflict✨

**Comment by kenos1** [comment]
"one more merge conflict and i will be switching careers"

File: src/config.ts [hunk 1]
```diff
@@ -104,20 +104,22 @@ type FishServer = {
 	port:string;
 	aliases:string[];
 	name:string;
+  requiredTrusted?:boolean;
```
Comment (BalaM314) [inline src/config.ts:107]: Change this to a string representing the Perm, and resolve it at usage location
Also, create a new perm called "hardcoreServer", and set it to trusted OR paying member

**Comment by BalaM314** [comment]
actually I need to fix the commits first

**Comment by BalaM314** [comment]
Oh great I broke it

**Comment by BalaM314** [comment]
I don't have access to kenos1/master because the PR is closed
the PR is closed because there are no commits on kenos1/master
there are no commits on kenos1/master because I don't have access to kenos1/master

Wonderful
Moved to #32

**Pull request closed without merging**